packages/subgraph/tests/cfav1/hol/cfav1.hol.test.ts

```diff
@@ -88,7 +91,15 @@ describe("ConstantFlowAgreementV1 Higher Order Level Entity Unit Tests", () => {
 
         handleFlowOperatorUpdated(flowOperatorUpdatedEvent);
 
-        
+        const approvalEvent = createApprovalEvent(
```

Comment:
Please remove this from here and put it in its own unit test in `supertoken/hol/supertoken.hol.test.ts`